### DISCUSSION TITLE
[Feature] notifications on new partner photos

### DISCUSSION THREAD

**Discussion by nreeb** [immich-app/immich#14882] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When a shared partner adds new photos it wild be nice to have an option to send a notification to ask other partners that the partner had uploaded new photos

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by stickflowerfightclub** [comment]
Same here ! It's an useful function in Google Photos. Should be awesome to have the same here.

**Comment by spiderben25** [comment]
Would love that too. I’ve recently migrated from Google Photos and so far it’s the only feature I’m missing, be able to receive a mobile notification whenever my partner adds a photo to her timeline.

**Comment by hunnguye** [comment]
Yes that would be great!
It doesn't need to be a full fledged push notification (yet). 
Actually just the notification / activity change which the WebUI shows in the bell icon would be enough for various cases e.g.:
* new asset uploaded into shared album
* comment on an asset (maybe later with @ taging other users?)
